server/src/realtime/health_test.go

```diff
@@ -0,0 +1,19 @@
+package realtime_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"scrumlr.io/server/realtime"
+)
+
+func TestRealtime_IsHealthy(t *testing.T) {
+	rt, err := realtime.NewNats("foo")
+	assert.NotNil(t, err)
+	assert.False(t, rt.IsHealthy())
+
+	rt, err = realtime.NewNats(SetupNatsContainer(t))
```

Comment:
A minor thing and no need to change it, but I personally like to test each state in a separate method so you can just see by the test name what's probably wrong without looking into the code. In this case that would mean that `ssert.False(t, rt.IsHealthy())` and  `assert.True(t, rt.IsHealthy())` would be part of two different test setups

Reply:
Well move it to two test cases

Reply:
Moved to table test.
```
=== RUN   TestBroker_IsHealthy
--- PASS: TestBroker_IsHealthy (0.33s)
=== RUN   TestBroker_IsHealthy/nats_client_has_has_wrong_url
    --- PASS: TestBroker_IsHealthy/nats_client_has_has_wrong_url (0.06s)
=== RUN   TestBroker_IsHealthy/nats_client_is_setup_correctly
    --- PASS: TestBroker_IsHealthy/nats_client_is_setup_correctly (0.27s)
PASS
```